app/javascript/components/App.js

```diff
@@ -67,19 +66,84 @@ export default class App extends React.Component {
       })
   }
 
-  createNewBook = (form) => {
-    console.log(form);
+  createNewBook = (newBook) => {
+    console.log(newBook)
```

Comment:
🔥  Don't want to leave console logs in our final code, so if this isn't needed, should be removed.